spec/mongoid/query_cache_spec.rb

```diff
@@ -123,6 +123,27 @@
         end
       end
     end
+
+    context "when cache is enabled and second batch of items is requested" do
+
+      before do
+        Mongoid::QueryCache.enabled = true
```

Comment:
Will the 'before do ... end' changes persist after the context scope is left? It doesn't seem to have any effect outside the context in which it is defined when I set it to false -- the default seems to be true from doing 'QueryCache.enabled?' elsewhere, so setting it to false would be a change.

It would probably be cleaner to add the after block, though capturing and reseting the original state seems awkward: can't just set it without knowing what the original setting was.